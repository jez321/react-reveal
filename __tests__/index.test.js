/*
 * index Test Suite
 *
 * Copyright Â© Roman Nosov 2017
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE.txt file in the root directory of this source tree.
 */
import { Fade, Flip, Rotate, Zoom } from '../';

import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import { configure, shallow } from 'enzyme';
import React from 'react';

configure({ adapter: new Adapter() });
//      <Reveal effect="some-effect" className="someClass" style={{ border: '1px solid red'}} props={{src: '/example.com'}}>
//    </Reveal>

describe('index', () => {
  it('renders a initial view', () => {
    const content = shallow(
      <div>
        <Zoom><div> Zoom Test test</div></Zoom>
        <Fade><div> Fade Test test</div></Fade>
        <Flip><div>Flip Test test</div></Flip>
        <Rotate><div> Rotate Test test</div></Rotate>
      </div>
    );
    expect(content.html()).toMatchSnapshot();
  });
});
