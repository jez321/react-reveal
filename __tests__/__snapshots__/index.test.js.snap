// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index renders a initial view 1`] = `"<div><div class="react-reveal"> Zoom Test test</div><div class="react-reveal"> Fade Test test</div><div class="react-reveal">Flip Test test</div><div class="react-reveal"> Rotate Test test</div></div>"`;
